shared/src/main/scala/amf/client/exported/config/ShapeRenderOptions.scala

```diff
@@ -23,3 +22,18 @@ case class ShapeRenderOptions(private[amf] val _internal: InternalShapeRenderOpt
 
   def withSchemaVersion(version: JSONSchemaVersion): ShapeRenderOptions = _internal.withSchemaVersion(version)
 }
+
+@JSExportAll
+@JSExportTopLevel("JSONSchemaVersions")
+object JSONSchemaVersions {
+  val UNSPECIFIED: JSONSchemaVersion   = Unspecified
+  val DRAFT_04: JSONSchemaVersion      = JsonSchemaDraft4
+  val DRAFT_07: JSONSchemaVersion      = JsonSchemaDraft7
+  val DRAFT_2019_09: JSONSchemaVersion = JsonSchemaDraft201909
+}
+
+sealed trait JSONSchemaVersion
```

Comment:
empty objects don't port correctly to Scala JS, maybe use a case object with a variable `name` or something?